demoapplication/src/main/java/com/strivacity/android/app/MainActivity.kt

```diff
@@ -147,10 +148,12 @@ fun Login(nativeSDK: NativeSDK) {
               coroutineScope.launch {
                 try {
                   val accessToken = nativeSDK.getAccessToken()
-                  println(nativeSDK.getAccessToken())
+                  println(accessToken)
```

Comment:
Realize this is a demo app, but we may want to comment this as potentially leaking a secret to logs in case someone does a copypaste.